experiment/simulation/js/bitwisefbit.js

```diff
@@ -0,0 +1,380 @@
+"use strict";
+const messages = [
+    "Hover over the 5-bit table and Select any two rows to add",
+    "Results are highlighted green and overflow is displayed in result table\nClick on 4-bit table to switch",
+    "Click on reset to clear and restart anytime",
+]
+let mat = [];
+let temp = [];
+let ToBeAdded = [];
+let SelectedRow = [];
+let ColoredIdx = [];
+let ResultArray = [0, 0, 0, 0, 0];
+let VisitedArray = [0, 0, 0, 0, 0];
+let ResultArrayname = ['rowbinar', 'row2s', 'row1s', 'rowunsigned', 'rowsigned'];
+let vis = [0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0];
+let SameElemFlag = 0;
```

Comment:
All variable names would start with a small letter and will be camelCase. Please correct them.